webapp/src/ts/services/feedback.service.ts

```diff
@@ -69,8 +73,13 @@ export class FeedbackService {
     this.LEVELS.forEach(level => {
       const original = this.options.console[level];
       this.options.console[level] = (...args) => {
+        const logEvent = {
+          level,
+          arguments: JSON.stringify(args, getCircularReplacer()).substring(0, 1000),
```

Comment:
Why this particular hard limit?

Reply:
Hey @m5r !  The 1000 char limit is [discussed here](https://github.com/medic/cht-core/pull/8164#discussion_r1160168017) and I welcome your input on it!